settings.gradle.kts

```diff
@@ -404,9 +404,9 @@ include(":instrumentation:restlet:restlet-2.0:library")
 include(":instrumentation:restlet:restlet-2.0:testing")
 include(":instrumentation:rmi:bootstrap")
 include(":instrumentation:rmi:javaagent")
-include(":instrumentation:rocketmq-client-4.8:javaagent")
-include(":instrumentation:rocketmq-client-4.8:library")
-include(":instrumentation:rocketmq-client-4.8:testing")
+include(":instrumentation:rocketmq:rocketmq-clients:rocketmq-client-4.8:javaagent")
```

Comment:
almost all other modules use singular name `rocketmq-client` instread of `rocketmq-clients`